funnel/models/space.py

```diff
@@ -115,7 +121,7 @@ def proposals_by_status(self):
             basequery = Proposal.query.filter(Proposal.proposal_space_id.in_([self.id] + [s.id for s in self.subspaces]))
         else:
             basequery = Proposal.query.filter_by(proposal_space=self)
-        return dict((status, basequery.filter_by(status=status).order_by(db.desc('created_at')).all()) for (status, title) in PROPOSALSTATUS.items() if status != PROPOSALSTATUS.DRAFT)
+        return dict((status, basequery.filter_by(_status=status).order_by(db.desc('created_at')).all()) for (status, title) in PROPOSALSTATUS.items() if status != PROPOSALSTATUS.DRAFT)
```

Comment:
This multiple query thing is a bad idea. It needs to go away.

1. In `PROPOSALSTATUS` add a group for all values that are not `DRAFT`.
2. Use [SQLAlchemy's `group_by`](http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.group_by) on the state column.
3. Turn the groups into a dictionary.

Reply:
I was going to ask you about this. that multiple query was bothering me as well.